modules/ROOT/pages/_partials/keycloak/setting-up-the-idm-sdk.adoc

```diff
@@ -158,12 +158,12 @@ Cordova::
 +
 --
 // end::excludeDownstream[]
-Import and initialize Auth:
 
+. Import and initialize Auth.
+. Replace `import` with `require` in the `init` section, as displayed in the following snippet:
++
 [source,javascript]
 ----
-const Auth = require('@aerogear/auth').Auth;
```

Comment:
I wouldn't remove this line

Reply:
Hi @austincunningham - I'm confused. Above you stated that you "don't think this line is needed"[1] yet after I removed it, you state that you wouldn't remove it. Would you mind making the correct technical update so we can progress this situation? cc @pnagesha for visibility.

[1] https://github.com/aerogear/mobile-docs/pull/676#discussion_r283759272

Reply:
#676 refers to the line 163

Reply:
Ah, ok @austincunningham :+1: 

So, as a Technical Writer, can I assume that the code is now correct? If not, can you please fix the code so we can progress this.

Reply:
@jstaffor you still need to add back in line 165 which content is as follows
```js
const Auth = require('@aerogear/auth').Auth;
```

Reply:
@austincunningham - I've reverted this change: https://github.com/aerogear/mobile-docs/pull/676/commits/e7e824a8951408c2d0b254490085a1df489194e1